### PR TITLE
ui content change

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: page/sass/commit_funds.scss [hunk 1]
```diff
@@ -47,13 +47,14 @@
       }
       .section-seperator{
         border-bottom: 1px $gray-extra-light solid;
-        width: 407px;
+        width: 90%;
         margin-left: 50px;
```
Comment (MostafaBalata) [inline page/sass/commit_funds.scss]: Can you create new variable for this space in the variables.scss?

Comment (MostafaBalata) [inline page/sass/commit_funds.scss]: Can you create new variable for this number in the variable.scss file?

File: page/sass/commit_funds.scss [hunk 2]
```diff
@@ -47,13 +47,14 @@
...
       }
       li{
         margin-left: 20px;
```
Comment (MostafaBalata) [inline page/sass/commit_funds.scss]: This number exists already in the variable.scss file, can you use it?

File: page/sass/common.scss [hunk 3]
```diff
@@ -12,7 +12,7 @@ h3.smaller {
 }
 
 .section ul.compact-list {
-    padding: 0 0 $space-bottom-between-elements 0;    
+    padding: 0 0 $space-bottom-between-elements 0;
 
     li {
         font-size: 16px;
```
Comment (MostafaBalata) [inline page/sass/common.scss]: And here, the number should be from the variables.scss file

File: page/sass/commit_funds.scss [hunk 4]
```diff
@@ -47,13 +47,14 @@
       }
       .section-seperator{
         border-bottom: 1px $gray-extra-light solid;
-        width: 407px;
+        width: 90%;
```
Comment (MostafaBalata) [inline page/sass/commit_funds.scss]: I think this is not the best way, to have this `.section-separator` fit to his parent, you can just solve the problem using html only, by moving this into the previous `div`, like this:
```
                <div class="content spacing">
                    <h3><%- sections[0].title -%></h3>
                        <p><%- sections[0].paragraph1 -%></p>
                        <p><%- sections[0].paragraph2 %></p>
                        <div class="section-seperator"></div>
                </div>
```
and then no need for `width: 90%`

🗡

Comment (Moejoe90) [reply page/sass/commit_funds.scss]: That's a good idea

File: page/sass/common.scss (hunk 3 above)
Comment (Moejoe90) [reply page/sass/common.scss]: It's not part of the pull request but good observation

File: page/sass/commit_funds.scss (hunk 2 above)
Comment (Moejoe90) [reply page/sass/commit_funds.scss]: This will be taken off

File: page/sass/commit_funds.scss (hunk 1 above)
Comment (Moejoe90) [reply page/sass/commit_funds.scss]: This will be taken off

File: page/partials/header.ejs [hunk 5]
```diff
@@ -26,4 +26,4 @@
       </ul>
     </div>
   </div>
-</nav>
\ No newline at end of file
+</nav>
```
Comment (Moejoe90) [inline page/partials/header.ejs]: Do we have linting rules regarding white spaces?

**Pull request merged**